gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -185,6 +189,31 @@ fixObject(OMR_VMThread *omrVMThread, MM_HeapRegionDescriptor *region, omrobjectp
 	}
 }
 
+static void
+fixFreeObject(OMR_VMThread *omrVMThread, MM_HeapRegionDescriptor *region, omrobjectptr_t object, void *userData)
+{
+	// OMRPORT_ACCESS_FROM_OMRVMTHREAD(omrVMThread);
+	MM_GCExtensionsBase *extensions = MM_GCExtensionsBase::getExtensions(omrVMThread->_vm);
+	// MM_ParallelGlobalGC *collector = (MM_ParallelGlobalGC *)extensions->getGlobalCollector();
+
+	// omrtty_printf("Current object address: %p\n", (void*)object);
+	// omrtty_printf("Current consumed size: %d\n", extensions->objectModel.getConsumedSizeInBytesWithHeader(object));
+	if (extensions->objectModel.isDeadObject(object)) {
+		if (extensions->objectModel.isSingleSlotDeadObject(object)) {
+			((memorySizes*) userData)->freeBytes += extensions->objectModel.getSizeInBytesSingleSlotDeadObject(object);
```

Comment:
in the cast operatior add space between the type name and '*' operator (here and elsewhere)
no need to spece between closing ')' of the cast operator and the variable userData